### DISCUSSION TITLE
Auth with JWT and Identity

### DISCUSSION THREAD

**Discussion by roman-koshchei** [stphnwlsh/CleanMinimalApi#18] [Ideas]
It will be cool if auth is included into such template. Because it's very often functional for modern applications.

**Comment by stphnwlsh** [comment]
Hey thanks for the suggestion!  Honestly I left it out because there's so many variables with auth that it would be difficult to meat everyone's needs.  I am updating the solution to .net 7 at the moment so I might take another look and see if I can implement a simple auth flow with JWT.  WIll let you know what I get up to
